R/compute.R

```diff
@@ -265,7 +269,14 @@ create_aks_compute <- function(workspace,
                                subnet_name = NULL,
                                service_cidr = NULL,
                                dns_service_ip = NULL,
-                               docker_bridge_cidr = NULL) {
+                               docker_bridge_cidr = NULL,
+                               cluster_purpose = NULL) {
```

Comment:
can it be `cluster_purpose = c('DevTest', 'FastProd')` so user can tell what the valid values are from the function signature

Reply:
actually, since cluster_purpose is optional and I believe defaults to FastProd if no value is provided, would this work instead: `cluster_purpose = c("FastProd", "DevTest")`